docs/docusaurus/docs/cloud/setup/setup_cloud.md

```diff
@@ -0,0 +1,183 @@
+
+After you've tested GX Cloud features and functionality, you can connect to your Data Source and create Expectations that are specific to your business needs.
+
+Although you can use any Python Interpreter or script file to run Python code, Great Expectations (GX) recommends using Jupyter Notebook. Jupyter Notebook is included with OSS GX and is the best option for composing script files and running code.
+
+The example code is available in the [onboarding script repository](https://github.com/great-expectations/great_expectations/blob/develop/assets/scripts/gx_cloud/experimental/onboarding_script.py).
+
+## Prerequisites
+
+- An active [GX Cloud Beta Account](https://greatexpectations.io/cloud).
+
+## Prepare your environment
+
+1. Download and install the latest Python version. See [Active Python Releases](https://www.python.org/downloads/).
+
+2. Download and install pip. See the [pip documentation](https://pip.pypa.io/en/stable/cli/pip/).
+
+3. Run the following command in an empty base directory inside a Python virtual environment:
+
+    ```bash title="Terminal input"
+    pip install great_expectations
+    ```
+
+    It can take several minutes for the installation to complete.
+
+    If you've previously installed GX, run the following command to upgrade to the latest version:
+
+    ```bash title="Terminal input"
+    pip install great_expectations --upgrade
+    ```
+4. In Jupyter Notebook, run the following Python code to import the modules you'll use to test functionality:
+
+    ```python title="Jupyter Notebook"
+    import great_expectations as gx
+    import pandas as pd
+    import os
+    ```
+
+## Generate your user access token and copy your organization ID
+
+You'll need your user access token and organization ID to create your Data Context. Access tokens shouldn't be committed to version control software.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **Access tokens** pane, click **Create user access token**.
+
+3. Complete the following fields:
+
+    - **Token name** - Enter a name for the token that will help you quickly identify it.
+
+    - **Role** - Select **Admin**. For more information about the available roles, click **Information** (?) .
+
+4. Click **Create**.
+
+5. Copy the access token and store it in a secure location. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field and save it in the same location with your user access token.
+
+
+## Create a Data Context
+
+A Data Context provides the configurations and methods for GX Cloud components, including Data Sources, Expectations, Profilers, and Checkpoints.
+
+1. In Jupyter Notebook, copy the following code into a cell to instantiate the GX Cloud Data Context.
```

Comment:
We start many instructions with "In Jupyter Notebook." Would it be appropriate to move all the GX workflow steps with code underneath an umbrella section ("Build and execute your Cloud workflow in code" or something similar?) that issues a blanket: Start your notebook, and execute all of the following code in Jupyter Notebook?

(I'm not super opinionated on this, I just noted the repetition).

Reply:
@rachhouse we can never determine where a user will start a task, so providing information about where the task occurs is very helpful. Also, adding an umbrella heading will unnecessarily complicate the heading structure. I think the simple approach is best and the topic functions well as it is currently presented.